### DISCUSSION TITLE
Stacked Bar/ Column Chart by Species Over Time?

### DISCUSSION THREAD

**Discussion by AlaskanAstro** [mcguirepr89/BirdNET-Pi#1206] [Ideas]
Does anyone know if it's possible to export the data from birdnet and create a stacked column chart? Something visually similar to this:
![333293933-8901bae7-05f5-4d6a-acfc-8f93e2634d89](https://github.com/mcguirepr89/BirdNET-Pi/assets/112118866/4547ec06-6f10-4b32-a3e0-5563c413a077)
I think it would be a very neat way to display detected birds over the year.

**Comment by Nachtzuster** [comment]
The data is all saved in a SQLite db: `/home/<user>/BirdNET-Pi/scripts/birds.db` the ui uses this one.
There is also a 'raw' csv file: `/home/<bird>/BirdNET-Pi/BirdDB.txt`

the difference is that the SQLite file is updated if you delete a detection in the UI, while the raw csv file is not
